src/core_atmosphere/physics/mpas_atmphys_driver_sfclayer.F

```diff
@@ -903,10 +902,8 @@ subroutine sfclayer_to_MPAS(configs,sfc_input,diag_physics,its,ite)
     call mpas_pool_get_array(sfc_input,'xice',xice)
     do j = jts,jte
     do i = its,ite
-       if(xice(i).ge.xice_threshold .and. xice(i).le.1._RKIND) then
+       if(xice(i).ge.xice_threshold .and. xice(i).lt.1._RKIND) then
           br(i)     = br_p(i,j)*xice(i)     + (1._RKIND-xice(i))*br_sea(i,j)
-          flhc(i)   = flhc_p(i,j)*xice(i)   + (1._RKIND-xice(i))*flhc_sea(i,j)
-          flqc(i)   = flqc_p(i,j)*xice(i)   + (1._RKIND-xice(i))*flqc_sea(i,j)
```

Comment:
I'm struggling to see how this does not have consequences outside of the RUC LSM application. This is basically removing these lines from the general case and placing them in a RUC-only case statement in the seaice driver.

Reply:
Only RUC LSM is using flhc and flqc. The testing for HRRRv5 with the NoahMP confirms that it is only RUC-related change.

Reply:
These are being used in the [YSU schemes in the surface layer driver](https://github.com/MPAS-Dev/MPAS-Model/blob/41e9a3fb8ca6b9250a7405209a5c60996318409f/src/core_atmosphere/physics/mpas_atmphys_driver_sfclayer.F#L903). Or they are at least being sent into the schemes. I assume that's why the above code is in the surface layer driver.

Reply:
The only land/ice model that actually use flqc and flhc is RUC LSM. They could be produced be MYNN and other surface-layer schemes.